<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vector Field Explorer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="viewerLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <widget class="VFE::MainViewer" name="mainViewer" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>4</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="VFE::SliceViewer" name="sliceViewer" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>4</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="controlPanel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="controlPanelLayout">
       <item>
        <widget class="VFE::FeatureList" name="featureList" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="VFE::FeatureWidget" name="featureEditor" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>790</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuApp">
    <property name="title">
     <string>&amp;VFE</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <addaction name="actionRenderingOptions"/>
    <addaction name="separator"/>
    <addaction name="actionReloadShaders"/>
    <addaction name="actionStartBenchmark"/>
    <addaction name="actionScreenshot"/>
   </widget>
   <addaction name="menuApp"/>
   <addaction name="menuOptions"/>
  </widget>
  <action name="actionLoad">
   <property name="text">
    <string>&amp;Load vector field...</string>
   </property>
   <property name="toolTip">
    <string>Load Vector Field</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit the application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionReloadShaders">
   <property name="text">
    <string>&amp;Reload shaders</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionRenderingOptions">
   <property name="text">
    <string>&amp;Rendering options...</string>
   </property>
  </action>
  <action name="actionStartBenchmark">
   <property name="text">
    <string>Start &amp;benchmark</string>
   </property>
  </action>
  <action name="actionScreenshot">
   <property name="text">
    <string>Save &amp;screenshot</string>
   </property>
   <property name="toolTip">
    <string>Save a screenshot of the 3D viewer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VFE::MainViewer</class>
   <extends>QWidget</extends>
   <header>Gui/MainViewer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VFE::SliceViewer</class>
   <extends>QWidget</extends>
   <header>Gui/SliceViewer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VFE::FeatureWidget</class>
   <extends>QWidget</extends>
   <header>FeatureWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VFE::FeatureList</class>
   <extends>QWidget</extends>
   <header>FeatureList.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
